src/lib/api/search.ts

```
@@ -0,0 +1,181 @@
+  result: FinnhubSearchResult[];
+};
+
+export type StockWithWatchlistStatus = {
+  symbol: string;
+  name: string;
+  exchange: string;
+  type: string;
+  isInWatchlist: boolean;
+};
+
+const FINNHUB_BASE_URL = (import.meta as any).env?.VITE_FINNHUB_BASE_URL as
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)